src/components/navBar/navBar.js

```diff
@@ -104,6 +104,24 @@ function MdNavBar($mdAria, $mdTheming) {
         '<md-nav-ink-bar ng-hide="ctrl.mdNoInkBar"></md-nav-ink-bar>' +
       '</div>',
     link: function(scope, element, attrs, ctrl) {
+
+      ctrl.width = $window.innerWidth;
```

Comment:
Based on the test and lint failures, it appears that `$window` needs to be injected above into
```js
function MdNavBar($mdAria, $mdTheming) {
```